test/gov/verify_surfaced_error.sh

```diff
@@ -0,0 +1,96 @@
+
+echo "Testing node error surfacing to exchange"
+
+IBM_ADMIN_AUTH="IBM/ibmadmin:ibmadminpw"
+KEY_TEST_DIR="/tmp/keytest"
+export HZN_EXCHANGE_URL="${EXCH_APP_HOST}"
+
```

Comment:
I need you to verify with Nadim that the 3 preceding lines are not going to cause a problem with remote testing.

Reply:
Okayed by Nadim